src/solver/optimisation/adequacy_patch_csr/construct_problem_constraints_RHS.cpp

```diff
@@ -69,86 +61,82 @@ void setRHSnodeBalanceValue(const PROBLEME_HEBDO* ProblemeHebdo,
-                = hourlyCsrProblem.numberOfConstraintCsrAreaBalance.find(Area);
-            if (it != hourlyCsrProblem.numberOfConstraintCsrAreaBalance.end())
+            std::map<int, int>::iterator it = numberOfConstraintCsrAreaBalance.find(Area);
+            if (it != numberOfConstraintCsrAreaBalance.end())
             {
                 int Cnt = it->second;
-                ProblemeAResoudre.SecondMembre[Cnt] = hourlyCsrProblem.rhsAreaBalanceValues[Area];
+                problemeAResoudre_.SecondMembre[Cnt] = rhsAreaBalanceValues[Area];
                 logs.debug() << Cnt << ": Area Balance: RHS[" << Cnt
-                             << "] = " << ProblemeAResoudre.SecondMembre[Cnt]
+                             << "] = " << problemeAResoudre_.SecondMembre[Cnt]
                              << " (Area = " << Area << ")";
             }
         }
     }
 }
 
-void setRHSbindingConstraintsValue(const PROBLEME_HEBDO* ProblemeHebdo,
-                                   PROBLEME_ANTARES_A_RESOUDRE& ProblemeAResoudre,
-                                   const HourlyCSRProblem& hourlyCsrProblem)
+void HourlyCSRProblem::setRHSbindingConstraintsValue()
 {
-    int hour = hourlyCsrProblem.hourInWeekTriggeredCsr;
-    double csrSolverRelaxationRHS = ProblemeHebdo->adqPatchParams->ThresholdCSRVarBoundsRelaxation;
-    double* SecondMembre = ProblemeAResoudre.SecondMembre;
-    std::map<int, int> bingdingConstraintNumber
-        = hourlyCsrProblem.numberOfConstraintCsrHourlyBinding;
+    double csrSolverRelaxationRHS = problemeHebdo_->adqPatchParams->ThresholdCSRVarBoundsRelaxation;
+    double* SecondMembre = problemeAResoudre_.SecondMembre;
 
     // constraint:
     // user defined Binding constraints between transmission flows
     // and/or power generated from generating units.
-    for (int CntCouplante = 0; CntCouplante < ProblemeHebdo->NombreDeContraintesCouplantes;
+    for (int CntCouplante = 0; CntCouplante < problemeHebdo_->NombreDeContraintesCouplantes;
          CntCouplante++)
     {
-        if (bingdingConstraintNumber.find(CntCouplante) != bingdingConstraintNumber.end())
+        if (numberOfConstraintCsrHourlyBinding.find(CntCouplante)
+            != numberOfConstraintCsrHourlyBinding.end())
         {
             const CONTRAINTES_COUPLANTES* MatriceDesContraintesCouplantes
-                = ProblemeHebdo->MatriceDesContraintesCouplantes[CntCouplante];
+              = problemeHebdo_->MatriceDesContraintesCouplantes[CntCouplante];
 
-            int Cnt = bingdingConstraintNumber[CntCouplante];
+            int Cnt = numberOfConstraintCsrHourlyBinding[CntCouplante];
 
             // 1. The original RHS of bingding constraint
-            SecondMembre[Cnt]
-                = MatriceDesContraintesCouplantes->SecondMembreDeLaContrainteCouplante[hour];
+            SecondMembre[Cnt] = MatriceDesContraintesCouplantes
+                                  ->SecondMembreDeLaContrainteCouplante[hourInWeekTriggeredCsr];
```

Comment:
About **hourInWeekTriggeredCsr** : because here we obviously are in the context of CSR, we could shorten this name into  something like **triggeredHour**. It would remain expressive.